packages/react-core/src/elements/types.ts

```diff
@@ -30,21 +30,21 @@ export type BaseElementWithRef<
 >;
 
 type ListElementSubType = 'Ordered' | 'Unordered';
-type ListElementDisplayName = 'List' | `${ListElementSubType}List`;
+type ListElementDisplayName = `${ListElementSubType}List`;
 
-type TableElementSubType = 'Body' | 'Data' | 'Row' | 'Head' | 'Header';
+type TableElementSubType = 'Body' | 'DataCell' | 'Row' | 'Head' | 'Header';
```

Comment:
Updating to align as Table Data Cell per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td